neural_compressor/torch/algorithms/weight_only/rtn.py

```diff
@@ -130,15 +130,13 @@ def convert(
 
         if use_layer_wise:
             from neural_compressor.common.utils import DEFAULT_WORKSPACE
-            from neural_compressor.torch.algorithms.layer_wise.utils import get_path, load_module, register_weight_hooks
+            from neural_compressor.torch.algorithms.layer_wise.utils import get_path, load_module
```

Comment:
I saw changes was applied in RTN only.  I know current fast config scenario covers RTN only. but for layer-wise, both GPTQ and AutoRound are supported, do we need similar fix in these two?

Reply:
GPTQ/AutoRound is TODO, (will report issue when run quantizaiton on xpu)